src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -79,6 +79,9 @@ def schedule_organizations(timestamp: float | None = None, duration: int | None
                     organization_id=organization.id, teams__projectteam__project__isnull=False
                 ).values_list("user_id", flat=True)
             }
+            if not user_ids:
```

Comment:
You should also filter this list for `None` as passing a list that contains `None` into `user_service.get_many` will trigger an error.

Given that we're sending notifications to users with this, we could skip members that haven't completed their invitations. Invites could be excluded by using a `user_id__isnull=False` condition.